### PR TITLE
fix #666, add UI for stock crawler example

### LINKED ISSUE #666: [example] add a stock index crawler

### PR DESCRIPTION
**Pull request by clockfly**
[Review on Reviewable](https://reviewable.io/reviews/intel-hadoop/gearpump/693)

### CODE REVIEW

File: examples/streaming/stockcrawler/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+How to use
+===================
+1. Start local cluster, 
+  ```
+  bin/local -ip 127.0.0.1 -port 3000
+  ```
+2. Submit the stock crawler
+  ```
+  bin\gear app -jar examples\gearpump-examples-assembly-0.3.2-SNAPSHOT.jar org.apache.gearpump.streaming.examples.stock.main.Stock -master 127.0.0.1:3000
```
Comment (stanleyxu2005) [inline examples/streaming/stockcrawler/README.md]: Do you think including a specified version number will not work for future version? Usually I'd use `examples\gearpump-examples-assembly-*.jar`

Comment (clockfly) [inline examples/streaming/stockcrawler/README.md]: OK, I can fix the comments later.

**Comment by clockfly** [comment]
ui

**Comment by stanleyxu2005** [comment]
+1

**Pull request merged**